sql-bricks.js

```diff
@@ -938,19 +938,33 @@
   sql._handleTables = handleTables;
 
   function handleTable(table, opts) {
-    return handleColumn(expandAlias(table), opts);
+    if (typeof table === 'string') {
+      return sql._prepareTableIdentifier(table, opts);
+    }
+    return prepareTableIdentifier(table, opts);
+  }
+
+  function prepareTableIdentifier(table, opts) {
+    return prepareColumnIdentifier(expandAlias(table), opts);

```

Comment:
This intentionally points to `prepareColumnIdentifier` rather than `sql._prepareColumnIdentifier` so that changing the column escaping behavior doesn't incidentally change the table escaping behavior.